cvat/apps/webhooks/permissions.py

```diff
@@ -108,9 +112,9 @@ def get_resource(self):
                     {
                         "owner": (
                             {
-                                "id": project.owner.id,
+                                "id": project.owner_id,
                             }
-                            if project.owner
+                            if project.owner_id
```

Comment:
```suggestion
                            if project.owner_id is not None
```

Reply:
`_id`s are either positive numbers or `None`, this change doesn't seem to add much. Do you suggest that this check patter is updated everywhere?